examples/openai/azure_openai.py

```diff
@@ -21,13 +23,13 @@
         task=openai.ChatCompletion,
         artifact_path="model",
         messages=[{"role": "user", "content": "Tell me a joke about {animal}."}],
-        deployment_id="<YOUR AZURE OPENAI DEPLOYMENT ID (ALSO CALLED DEPLOYMENT NAME)>",
+        deployment_id=os.environ["DEPLOYMENT_ID"],
     )
 
 # Load native OpenAI model
 native_model = mlflow.openai.load_model(model_info.model_uri)
 completion = openai.ChatCompletion.create(
-    model=native_model["model"],
+    deployment_id=os.environ["DEPLOYMENT_ID"],
```

Comment:
Does `native_model` contain `deployment_id`?

Reply:
Ah it does!